ee/codegen/src/__test__/project.test.ts

```diff
@@ -48,6 +49,9 @@ describe("WorkflowProjectGenerator", () => {
           // TODO: Get Merge Node graph codegen working
           //    https://app.shortcut.com/vellum/story/5588
           // "simple_merge_node",
+
+          // Real customer workflows:
+          //"faa_q_and_a_bot",
```

Comment:
[1/5 opinion] I'd say we should probs exclude this fixture from this PR unless we're prepared to uncomment this and include as a test we test again